### ISSUE TITLE
Field Map Masks for Brain + C-Spine Images

### ISSUE THREAD

**Issue by Govish** [shimming-toolbox/susceptibility-to-fieldmap-fft#21]
Hi guys!
I took a peek at the masks you sent over, and it looks like they include just the brain.
Would it be possible to get masks that include all anatomy requiring shimming (i.e. brain + c-spine, etc.)?

Thanks!
-Ishaan

**Comment by CharlesPageot** [comment]
Hi @Govish! Sorry for the wait.
We have the whole spine segmented for the dataset. I will generate the masks.
Just to be sure, you want a **single mask** for each of the subject that include brain + spine ?

**Comment by Govish** [comment]
Hi Charles! 
Sorry for the radio silence on my end! I just got a chance to go through the masks you sent that included the spine, and they're a lot closer to what we're looking for. However, it looks like they include the **whole spine**. Would it be possible to get masks that **just include the brain + C-spine (C1-C7)**? 

Thanks for your continued help and support!
-Ishaan

**Comment by Govish** [comment]
^ In chatting with my PI's actually, would getting all segmented tissue maps be possible? (e.g. having separate masks for brain, all vertebrae of the spine, lungs, etc.)

**Comment by CharlesPageot** [comment]
Yeah, for sure, we have the combined label maps. The vertebrae are currently all under the same class (same thing for the discs). Do you need them to be all separate (i.e. one label for C3, another one for C4 etc) ?

**Comment by Govish** [comment]
That would be great! Apologies for the late reply!

**Comment by jcohenadad** [comment]
@Govish is this issue solved?

**Comment by Govish** [comment]
Not yet (at least I don’t have individual tissue maps yet)—I may have missed an email/message however.
Thanks for all the help!
